ros_ethercat_model/CMakeLists.txt

```diff
@@ -18,8 +18,6 @@ find_package(catkin REQUIRED COMPONENTS
 find_package(Boost REQUIRED COMPONENTS system thread)
 find_package(TinyXML2 REQUIRED)
 
-catkin_python_setup()
```

Comment:
Should we also remove https://github.com/shadow-robot/ros_ethercat/blob/melodic-devel/ros_ethercat_model/setup.py

Reply:
removed